### ISSUE TITLE
TextraLabel {SIZE=x} and {SCALE=x} does not work right

### ISSUE THREAD

**Issue by Trurl101** [tommyettinger/textratypist#20]
```
String text = "Hello, world!\nDid you know\norange is\nmy favorite color?";

final TextraLabel testLabel1 = new TextraLabel("{SCALE=50%}" + text, style);
testLabel1.setPosition(10, 500);
stage.addActor(testLabel1);

final TextraLabel testLabel2 = new TextraLabel(text, style);
testLabel2.setPosition(10 + testLabel1.getX() + testLabel1.layout.getWidth(), 500);
stage.addActor(testLabel2);

final TextraLabel testLabel3 = new TextraLabel("{SCALE=150%}" + text, style);
testLabel3.setPosition(10 + testLabel2.getX() + testLabel2.layout.getWidth(), 500);
stage.addActor(testLabel3);
```

Above code produce following labels:

<img width="625" alt="Bildschirmfoto 2024-11-28 um 21 29 58" src="https://github.com/user-attachments/assets/ebeae622-12b6-4450-9c1f-9a2b45ca8ae4">

Without scale (in the middle) the label is as expected. With scale the second line has a wrong Y position. Style has a simple bitmapfont created with freetype from a ttf, but its the same for all fonts I tried.

**Comment by tommyettinger** [comment]
I don't know why the first line break doesn't seem to respect sizing. If the first line was relatively higher, I'd think all of them would be correct, except that the smaller size is still using the line height of 100% size. I'll look into this... Thanks for providing a test case, also.

**Comment by tommyettinger** [comment]
I was in the middle of some sizing revamp, so a new bug isn't unexpected, but after 1.1.0 using the current code, it looks worse... 😭 
![java_KeQ4nebyR6](https://github.com/user-attachments/assets/585b9b4a-6b8e-4512-b0dc-3e7d023c7db8)

**Comment by tommyettinger** [comment]
It gets weirder. With debugAll on, I get:

![java_jlA09SeneJ](https://github.com/user-attachments/assets/8748a202-8bb0-4e2c-a87b-d763e6d35146)

It looks like the bounds are correctly sized for all of them, but it's drawing out of bounds, and everything changes y-position as it scales for some reason.

**Comment by tommyettinger** [comment]
Fixed! Now I just have everything else to fix, in existence, for the 1.2.0 release. Can't be too hard.

![java_tYjRpYDsr4](https://github.com/user-attachments/assets/b22c528e-4db7-4be8-a92a-2b5199b0c502)
